app/Http/Controllers/InterOp/UserGroupsController.php

```diff
@@ -39,16 +40,16 @@ public function setDefault()
 
     private function getUserAndGroupModels()
     {
-        $params = get_params(request()->all(), null, [
-            'group_id:int',
-            'user_id:int',
-        ]);
+        $params = array_merge(
+            ['group' => get_int(request()->input('group_id'))],
+            request()->route()->parameters(),
+        );
 
-        $group = app('groups')->byId($params['group_id'] ?? null);
+        $group = app('groups')->byId($params['group'] ?? null);
```

Comment:
`$params['group']` may be string (also `?? null` isn't needed; similarly for `user` below)

Though looking again, `byId` shouldn't be restricted to `?int` in the first place to match `find()` on models (or it should cast to int inside the function).